lib/list.gd

```diff
@@ -1002,6 +1002,34 @@ DeclareOperation( "PositionsProperty", [ IsList, IsFunction ] );
 DeclareOperation( "PositionBound", [ IsList ] );
 
 
+#############################################################################
+##
+#O  PositionsBound( <list> ) . . . . position of first bound element in a list
+##
+##  <#GAPDoc Label="PositionsBound">
+##  <ManSection>
+##  <Oper Name="PositionsBound" Arg='list'/>
```

Comment:
This function will not turn up in the manual unless you explicitly include it in one of the `doc/ref/*.xml` files. It would hence stay undocumented for endusers.